bot/src/commands/revert-last-command.js

```diff
@@ -0,0 +1,54 @@
+import { get, post } from '../utils';
+import { Commands } from './index';
+
+function getCommandByName(name) {
+  for (let Command of Commands) {
+    if (Command.name === name) {
+      return Command;
+    }
+  }
+  return null;
+}
+
+export const RevertLastCommand = {
+  name: 'revert-last-command',
+
+  matches(str) {
+    return str === 'scratch that';
+  },
+
+  execute(bot, author, channel, text) {
+    const log = bot.getActionLog(author, channel);
+    if (log.length === 0) {
+      return Promise.resolve('There is no command to revert');
+    }
+    const entry = log[log.length - 1];
```

Comment:
Can you do `const entry = log.pop();` here? Do we need to keep the log item around?